packages/framework/src/Model/Localization/Localization.php

```diff
@@ -42,12 +43,19 @@ class Localization
      */
     private $allLocales;
 
+    /**
+     * @var string
+     */
+    protected $adminLocale;
+
     /**
      * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     * @param string $adminLocale
      */
-    public function __construct(Domain $domain)
+    public function __construct(Domain $domain, $adminLocale = 'en')
```

Comment:
Consider type-hinting it with `string`. Why having a default value duplicate with setting in `services.yml`?

Reply:
Because I wanted to avoid a bc break for people that might already extend Localization class and call `parent::__construct`.

Reply:
If you're trying to avoid BC breaks you must not add type-hints for return values.

Reply:
Right. As we consult, I added type-hints everywhere in this class and removed this default value as we're still in beta and we want to make our codebase cleaner as much as possible before stable. 
Upgrade notes were updated accordingly.